### PR TITLE
Add functionality for Exporter headers env-var #204

### PR DESCRIPTION
**Pull request by bryce-b**

### CODE REVIEW

**Comment by bryce-b** [comment]
I did some research on env-vars in Xcode after a hunch I got looking at the way env-vars are set in Xcode (only possible through the `run` option). Env-vars will only apply to app state ran though xcode, and wont be transmitted to archive builds. It seems obvious after the fact. I think that we may need another method of configuration akin to Env-var config available in other sdks. 
@nachoBonafonte do you think we should allow these configurations to be set up through plists?

File: Sources/OpenTelemetrySdk/Trace/TracerProviderBuilder.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        spanProcessors.append(spanProcessor)
+        return self
+    }
+
+    public func add(spanProcessors: [SpanProcessor]) -> Self {
+        self.spanProcessors.append(contentsOf: spanProcessors)
+        return self
+    }
+
+    public func build() -> TracerProvider {
```
Comment (nachoBonafonte) [inline Sources/OpenTelemetrySdk/Trace/TracerProviderBuilder.swift:53]: we could return here TracerProviderSdk

Comment (nachoBonafonte) [inline Sources/OpenTelemetrySdk/Trace/TracerProviderBuilder.swift:53]:
```suggestion
    public func build() -> TracerProviderSdk {
```
we could return here TracerProviderSdk

**Review by nachoBonafonte — APPROVED** [review]
It looks really good to me

**Pull request merged**